mockData/mockData.json

```diff
@@ -0,0 +1,114 @@
+[
+  {
+    "merchant": "Safeway",
+    "date": "03/25/2023",
+    "subtotal": 45.00,
+    "time": "14:30:45",
+    "tax_percentage": 6.33,
+    "tax": 2.85,
+    "total_amount": 47.85,
+    "payment_method": "Credit Card",
+    "card_type": "Visa",
+    "card_number": "**** **** **** 1234",
```

Comment:
These are also not included in our design spec, I think we should only use fields that we plan on using